README.md

```diff
@@ -0,0 +1,96 @@
+# @dojo/framework
```

Comment:
scrap my comments on this, we can fix this up in a follow up pr before releasing.